.github/workflows/pkgci_regression_test_amdgpu_vulkan.yml

```diff
@@ -57,7 +57,7 @@ jobs:
         uses: actions/checkout@8f4b7f84864484a7bf31766abe9204da3cbe65b3 # v3.5.0
         with:
           repository: nod-ai/SHARK-TestSuite
-          ref: 46ab63b271fddd78e071c0590e9a073f6282b005
+          ref: e266c91e03734fe3148f269b02ff4b9e4bfd5df9
```

Comment:
Hah, future-me gets to debug this: https://github.com/openxla/iree/actions/runs/8303455114/job/22727749721?pr=16801#step:9:51

```
Stderr diagnostics:
opt-125M.mlirbc:1:1: error: custom op 'version' is unknown (tried 'builtin.version' as well)
version https://git-lfs.github.com/spec/v1
^
```

(not trying to use git LFS, so need to check that the file is a real file and not just a reference)